utils.py

```diff
@@ -113,7 +113,9 @@ def get_build():
 
 def print_red(msg):
     print("\033[91m%s\033[0m" % msg, file=sys.stderr)
+    sys.stderr.flush()
```

Comment:
stderr is unbuffered and many applications leverage this property. No need for flush.

Reply:
Indeed, see also `man 3 stderr`:

> The  stream  stderr  is  unbuffered.

Reply:
Though note that it seems that @kees is modifying the buffering of stderr for our boot wrapper script in https://github.com/ClangBuiltLinux/boot-utils/pull/49.